packages/peregrine/lib/Price/__docs__/price.md

```diff
@@ -7,7 +7,7 @@ Formatting of prices and currency symbol selection is handled entirely by the EC
 ## Usage
 
 ```jsx
-import Price from '@peregrine/Price';
+import Price from '@magento/peregrine/lib/components/Price';
```

Comment:
Should we encourage named exports so that people can do `import { Price } from '@magento/peregrine'`

Reply:
Eventually. That relies on Webpack tree shaking, which as of right now is less reliable than just direct path imports.

Reply:
We literally just upgraded `react-feather` (in #1416) to a version that allows us to import from the root module rather than digging into its directory structure, and I was glad to merge it. Couldn't we do the same thing?